### DISCUSSION TITLE
Need more global settings for the Advanced Table

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-blocks#1194] [Feature Requests]
A lot of the key settings for the table need to be applied 1-by-1 
<img width="285" alt="Screen Shot 2020-12-18 at 11 24 21 AM" src="https://user-images.githubusercontent.com/3868028/102637195-a1d9d300-4123-11eb-99f2-6a863298b8d1.png">

This makes it very difficult to add borders or style to the table

**Comment by htmgarcia** [comment]
@stevejburge agree.
